Homework_Arrays/Task3_6.java

```diff
@@ -0,0 +1,20 @@
+package Homework_Arrays;
+import java.util.Scanner;
+public class Task3_6 {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter the first array, devide the elements with coma");
+        String input1 = scanner.nextLine();
+
+        System.out.println("Enter the second array, devide the elements with coma");
+        String input2 = scanner.nextLine();
+
+        if (input1.equals(input2)) {
```

Comment:
Using == on array will not give the desired result and it will compare them as objects

Please check slide 13 from Arrays lecture.

A simple way is to run a loop and compare elements one by one. 
Java provides a direct method Arrays.equals() to compare two arrays. 